bluesky_widgets/qt/search.py

```diff
@@ -0,0 +1,5 @@
+from ._run_tree import *
```

Comment:
The `run_tree` is its own thing, not related to search. Should remain a public module under `qt`.